engine/model_helpers.go

```diff
@@ -2720,8 +2720,11 @@ func (tps TPAttributes) AsTPAttributes() (result []*utils.TPAttributeProfile) {
 				th.FilterIDs = append(th.FilterIDs, filter)
 			}
 		}
-		if tp.Context != "" {
-			th.Context = tp.Context
+		if tp.Contexts != "" {
+			contextSplit := strings.Split(tp.Contexts, utils.INFIELD_SEP)
+			for _, context := range contextSplit {
+				th.Contexts = append(th.Contexts, context)
```

Comment:
append here can be more efficient without for, ie: th.Contexts = append(th.Contexts, contextSplit...)